packages/components-lib/src/hooks/useGeocoder.ts

```diff
@@ -216,11 +164,11 @@ const useGeocoder = (props: GeocoderProps) => {
         JSON.stringify(recentlySearchedItems),
       );
 
-      fly(feature);
+      flyTo(feature);
 
       close();
     },
-    [props.localStorageRecentlySearchedIemskey, props.language, fly, close],
+    [props.localStorageRecentlySearchedIemskey, props.language, flyTo, close],
```

Comment:
fly is a function outside of reacts render cycle, so no need to add it inside the depds, and hence no need for binding it with a flyTo callback